lib/swoosh/adapters/postmark.ex

```diff
@@ -39,15 +40,24 @@ defmodule Swoosh.Adapters.Postmark do
     end
   end
 
-  defp base_url(config), do: config[:base_url] || @base_url
-
   defp prepare_headers(config) do
     [{"User-Agent", "swoosh/#{Swoosh.version}"},
      {"X-Postmark-Server-Token", config[:api_key]},
      {"Content-Type", "application/json"},
      {"Accept", "application/json"}]
   end
 
+  defp prepare_url(config, email),
+    do: [base_url(config), api_endpoint(email)]
+
+  defp base_url(config),
+    do: config[:base_url] || @base_url
+
+  defp api_endpoint(%{"TemplateModel" => _, "TemplateId" => _}),
```

Comment:
Which means you'd get the `%Email{}` struct here.

That way it's consistent with the rest of the file.